src/main.ts

```diff
@@ -2,7 +2,7 @@ import { NestFactory } from '@nestjs/core';
 import { ExportModule } from './export.module';
 
 export async function bootstrap() {
-  const app = await NestFactory.create(ExportModule);
+  const app = await NestFactory.create(ExportModule, { cors: true });
```

Comment:
Why do we want to enable CORS ? Shouldn't our domain remain same for UI and backend servers

Reply:
We have CORS for all services; port matters also

Reply:
I thought we had CORS for all our services (Maybe I'm not remembering correctly).